posthog/models/event/sql.py

```diff
@@ -354,11 +354,22 @@
 """
 
 GET_EVENTS_VOLUME = "SELECT event, count() AS count, max(timestamp) AS last_seen_at FROM events WHERE team_id = %(team_id)s AND timestamp > %(timestamp)s GROUP BY event ORDER BY count DESC"
-
-GET_TOTAL_EVENTS_VOLUME = "SELECT count() AS count FROM events WHERE team_id = %(team_id)s"
+GET_EVENT_PROPERTY_SAMPLE_JSON_VALUES = """
```

Comment:
I need to check this on the laptop but I believe this will have awful performance on clickhouse, potentially slowing down other queries which need to run at the same time.

Reply:
That's a good point, checked this on Cloud for our prod team and this does take a good 10 s, so it seems better to run this _only_ in the demo. Refactored the function so that it works the same way in production as currently, and the extra analysis is enabled with `complete_inference=True`

Reply:
:bow: Thanks for handling.